generic_sensor_data_update_rate: {
  TACTILE_SENSOR_TYPE_SAMPLE_FREQUENCY_HZ: -2.,
  TACTILE_SENSOR_TYPE_MANUFACTURER: -2.,
  TACTILE_SENSOR_TYPE_SERIAL_NUMBER: -2.,
  TACTILE_SENSOR_TYPE_SOFTWARE_VERSION: -2.,
  TACTILE_SENSOR_TYPE_PCB_VERSION: -2.,
  TACTILE_SENSOR_TYPE_WHICH_SENSORS: -2.
  }

pst3_sensor_data_update_rate: {
  TACTILE_SENSOR_TYPE_PST3_PRESSURE_TEMPERATURE: -1.,
  TACTILE_SENSOR_TYPE_PST3_PRESSURE_RAW_ZERO_TRACKING: 1.,
  TACTILE_SENSOR_TYPE_PST3_DAC_VALUE: 1.
  }

biotac_sensor_data_update_rate: {
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_1: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_2: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_3: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_4: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_5: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_6: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_7: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_8: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_9: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_10: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_11: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_12: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_13: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_14: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_15: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_16: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_17: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_18: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_ELECTRODE_19: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_PDC: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_TAC: -1.,
  TACTILE_SENSOR_TYPE_BIOTAC_TDC: -1.
  }